driver-sync/src/main/com/mongodb/client/internal/MongoClientDelegate.java

```diff
@@ -71,12 +74,28 @@ public OperationExecutor getOperationExecutor() {
     }
 
     @Nullable
-    public ClientSession createClientSession(final ClientSessionOptions options) {
+    public ClientSession createClientSession(final ClientSessionOptions options, final ReadConcern readConcern,
```

Comment:
Do we need this method - could we pre-merge the readConcern and writeConcern, before hitting this method?

Reply:
I did it this way to keep DRY, since this is called from both com.mongodb.MongoClient and com.mongodb.client.MongoClientImpl.  

Note that this method, while public, is still in an internal class

Reply:
Ack